bower.json.erb

```diff
@@ -2,7 +2,7 @@
   "name": "rad-consumer",
   "dependencies": {
     "dough": "<%= gem_path('dough-ruby') %>",
-    "yeast": "1.10.0",
+    "yeast": "https://github.com/moneyadviceservice/yeast.git#10985_rad_MAPS-Banner",
```

Comment:
This will have to be updated to reflect the latest version of Yeast once it gets deployed